client/layout/index.js

```diff
@@ -124,6 +125,7 @@ export class PageLayout extends Component {
 		);
 	}
 }
+export const PageLayout = useFilters( PAGES_FILTER )( _PageLayout );
```

Comment:
Just curious here what the advantage of `useFilters` is. It appears to require `PAGES_FILTER` to be passed around with little benefit.

Reply:
That's so the `<PageLayout>` component (and so, the `<Router>`) is re-rendered when `useFilter(PAGES_FILTER, () => { ... })` is called anywhere. `useFilters` indicated that this component uses the result of `PAGES_FILTER`, so it needs to be re-rendered when that result can potentially change.

Reply:
Ah ok I see. Cool, could you add a comment to the code with a word or two about that?

Reply:
Sure, done in https://github.com/woocommerce/woocommerce-admin/pull/2492/commits/59f158f5738f61ebfadd9bc0d9a035794137868c